packages/@yoda/bluetooth/a2dp.js

```diff
@@ -169,7 +169,7 @@ BluetoothA2dp.prototype.handleEvent = function (data, mode) {
       var results = msg.results
       var nbr = results.deviceList != null ? results.deviceList.length : 0
       logger.debug(`Found ${nbr} devices, is_comp: ${results.is_completed}, currentDevice: ${results.currentDevice}`)
-      for (var i = 0; i < nbr; i++) {
+      for (i = 0; i < nbr; i++) {

```

Comment:
`var` statement is function scoped, then you declare a variable `i`, so that the CI reports that you redeclaration here. Remove the `var` is not the right way, I think creating scope by wrapping function is the only way to do.

Reply:
The simple variable "i" cannot be used twice? Even so, I think it just need to replace "i" with another variable "j".